Tests/TuistKitTests/Services/AccountUpdateUsernameTests.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Mockable
+import TuistLoader
+import TuistServer
+import TuistSupport
+import TuistSupportTesting
+
+@testable import TuistKit
+
+final class AccountUpdateUsernameServiceTests: TuistUnitTestCase {
```

Comment:
We're moving away from using `XCTest` in favor of the new [Testing](https://developer.apple.com/xcode/swift-testing/) library. You can find examples of that in our codebase if you search for `import Testing`